modules/agar/src/main/ts/ephox/agar/api/TestStore.ts

```diff
@@ -11,7 +13,8 @@ export interface TestStore {
   cClear: Chain<any, any>;
   sAssertEq: <T> (label: string, expected: any[]) => Step<T, T>;
   cAssertEq: <T> (label: string, expected: any[]) => Chain<T, T>;
-  assertEq: (label: string, expected: any[]) => void;
+  assertEq: (labely: string, expected: any[]) => void;
+  assertSortedEq: (labely: string, expected: any[]) => void;
```

Comment:
It's a bit weird having the types by any then have sorting on this since the sort function will not sort non primitive objects unless you have a sort function. Also why is this entire thing any wouldn't a type variable be useful here the expected would be an array of what you add?

- [ ] Consider using type variables instead of just adding a new module that allows any all over and has no type safety
- [ ] Consider adding an explicit sort comparator function so the type signature would be `assertSortedEq(label: string, expected: T[], cmp: (a: T, b: T): number): void;`

Reply:
Added types and comparator function.